src/components/context/SettingsContext.tsx

```diff
@@ -0,0 +1,113 @@
+import _ from 'lodash';
```

Comment:
We usually don't import the full lodash, but rather just the function you want to use. https://www.npmjs.com/package/lodash.countby